pkg/networkserver/grpc_asns.go

```diff
@@ -99,10 +100,17 @@ func (ns *NetworkServer) processApplicationUplinkTask(ctx context.Context, consu
 		}
 
 		cl := ttnpb.NewNsAsClient(conn)
+		var sendErr bool
```

Comment:
Heavy Nit: `sendErr` sounds like an error that needs to be sent, like we use `retErr` for errors to be returned.